components/Modal.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+import FontAwesome from 'react-fontawesome';
+
+class ModalExample extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      modal: false,
+    };
+
+    this.toggle = this.toggle.bind(this);
+  }
+
+  toggle() {
+    this.setState({
+      modal: !this.state.modal,
+    });
+  }
+
+  render() {
+    return (
+      <div className="align-middle">
+        <Button
+          color="warning"
+          onClick={this.toggle}
+          className="p-0 pr-1 pl-1 ml-2 mt-3"
+          style={{ borderRadius: '70px' }}
+        >
+          <FontAwesome name="info-circle" />
+        </Button>
+        <Modal isOpen={this.state.modal} toggle={this.toggle}>
+          <ModalHeader toggle={this.toggle}>
+            Traffic Crashes Information
+          </ModalHeader>
+          <ModalBody>
+            <p>
+              The information in this map is for the time period July 1, 2014
+              through three months from the present. We will upload new data
+              each month, always three months behind to allow for data
+              verification.
+            </p>
+            <p>
+              For the purposes of this map, "crash" is defined as a traffic
+              incident in the City of Boston that resulted in a response by
+              Boston Emergency Medical Services. These include incidents on
+              streets owned by the City, as well as incidents on state-owned
+              roadways. Note the following:
+            </p>
+            <p>
```

Comment:
Probably switch to a `<ol>` for this numbered list.

Reply:
👍